server/src/test/java/org/elasticsearch/search/aggregations/bucket/TermsTests.java

```diff
@@ -168,64 +167,52 @@ private List<BucketOrder> randomOrder() {
 
     public void testSupportsParallelCollection() {
         {
-            AggregatorFactories.Builder builder = new AggregatorFactories.Builder();
```

Comment:
I realized that I don't need to test AggregatorFactories.Builder here, hence I simplified the test while I was at it.